docs/modules/getting-started/pages/blue-green.adoc

```diff
@@ -0,0 +1,255 @@
+= Deploy Blue-Green Clusters
+:description: This tutorial guides you on how you can use Hazelcast's blue-green deployment feature for your client to operate continuously when its connected cluster is down.
+
+{description} The feature provides a client failover mechanism for rerouting the client connections to a different cluster without requiring a
+client network configuration update or client restart. You can use it to manually redirect client traffic in order to perform maintenance or software updates,
+as well as automatically redirect client traffic to a different cluster during a disaster recovery scenario.
+
+This tutorial focuses on the disaster recovery scenario, and you will:
+
+. Start an Enterprise cluster (blue)
+. Start another Enterprise cluster (green)
+. Configure a Java client to have the Blue-Green deployment's failover feature
+. Connect this client to the first cluster
+. Kill the first cluster and see that the client connects to the second one
+
+NOTE: Blue-Green deployment is currently supported by Hazelcast's Java, Node.js and Go clients.
+
+== Before You Begin
+
+To complete this tutorial, you need the following:
+
+[cols="1a,1a"]
+|===
+|Prerequisites|Useful resources
+
+|Docker
+|
+link:https://docs.docker.com/docker-for-mac/install/[Install Docker for Mac]
+
+link:https://docs.docker.com/engine/install/[Install Docker for Linux]
+
+link:https://docs.docker.com/docker-for-windows/install/[Install Docker for Windows]
+
+|Docker image for the full Hazelcast distribution
+|xref:get-started-enterprise.adoc[Install Hazelcast Enterprise]
+
+|Docker network with the name `hazelcast-network`
+|Use the `docker network create hazelcast-network` command 
+
+|===
+
+== Step 1. Start an Enterprise Cluster (Blue)
+
+In this step, you use the Hazelcast Enterprise Docker image to start a local single-member cluster called `blue`.
+This step also installs your Enterprise license key.
+
+Run the following command on the terminal:
+
+[source,shell]
+----
+docker run \
+    --network hazelcast-network \
+    --rm \
+    -e HZ_CLUSTERNAME=blue \
+    -e HZ_LICENSEKEY=<your license key> \
```

Comment:
Wherever we use placeholders `<...>` we need to call them out and remind users to replace them with actual data.